src/pymorize/data_request/variable.py

```diff
@@ -342,6 +343,13 @@ def attrs(self) -> dict:
             "cell_measures": self.cell_measures,
         }
 
+    def get_global_attributes(self):
+        return self.ga.get_global_attributes(self.rule_attrs, self.table_header)
+
+    def set_global_attributes(self, cv, rule_attrs):
+        self.ga = GlobalAttributes(cv)
+        self.rule_attrs = rule_attrs
+
```

Comment:
I think this can be removed

```suggestion

    def set_global_attributes(self, cv, rule_attrs):
        self.ga = GlobalAttributes(cv)
        self.rule_attrs = rule_attrs

```

Reply:
and also working on this now